thrift/zipkinCore.thrift

```diff
@@ -198,6 +197,26 @@ const string HTTP_RESPONSE_SIZE = "http.response.size"
  */
 const string LOCAL_COMPONENT = "lc"
 
+#***** Annotation.value or BinaryAnnotation.key ******
+/**
+ * When an annotation value, this indicates when an error occurred. When a
+ * binary annotation key, the value is a human readable message associated
+ * with an error.
+ *
+ * Due to transient errors, an ERROR annotation should not be interpreted
+ * as a span failure, even the annotation might explain additional latency.
+ * Instrumentation add the ERROR binary annotation is added when the operation
```

Comment:
grammar.. perhaps "Instrumentation should add the ERROR binary annotation when the operation failed and couldn't be recovered."